pkg/operator/controllers/routefix/routefix.go

```diff
@@ -11,25 +11,63 @@ import (
 	securityv1 "github.com/openshift/api/security/v1"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/kubernetes/pkg/apis/rbac"
 
 	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
 	"github.com/Azure/ARO-RP/pkg/util/version"
 )
 
 const (
-	kubeName           = "routefix"
-	kubeNamespace      = "openshift-azure-routefix"
-	kubeServiceAccount = "system:serviceaccount:" + kubeNamespace + ":default"
-	shellScript        = `for ((;;))
+	kubeName            = "routefix"
+	kubeNamespace       = "openshift-azure-routefix"
+	kubeServiceAccount  = "system:serviceaccount:" + kubeNamespace + ":default"
+	containerNameDrop   = "drop-icmp"
+	containerNameDetect = "detect"
+	shellScriptLog      = `while true;
 do
-  if ip route show cache | grep -q 'mtu 1450'; then
-    ip route show cache
-    ip route flush cache
-  fi
-  sleep 60
+	NOW=$(date "+%Y-%m-%d %H:%M:%S")
+	DROPPED_PACKETS=$(ovs-ofctl -O OpenFlow13 dump-flows br0 | sed -ne '/table=10,.* actions=drop/ { s/.* n_packets=//; s/,.*//; p }')
+	if [ "$DROPPED_PACKETS" != "" ] && [ "$DROPPED_PACKETS" -gt 1000 ];
+	then
+		echo "$NOW table=10 actions=drop packets=$DROPPED_PACKETS broken=true"
+	else
+		echo "$NOW table=10 actions=drop packets=$DROPPED_PACKETS broken=false"
+	fi
+	sleep 60
 done`
+	shellScriptDrop = `set -xe
+if [[ -f "/env/_master" ]]; then
+	set -o allexport
+	source "/env/_master"
+	set +o allexport
+fi
```

Comment:
what's this for?

Reply:
Intent is to source something on the host.  I don't see this file so it won't be run.  I'll remove in my local test and confirm we don't need it.